egs/tedlium3/ASR/RESULTS.md

```diff
@@ -0,0 +1,69 @@
+## Results
+
+### TedLium3 BPE training results (Transducer)
+
+#### Conformer encoder + embedding decoder
+
+Using the codes from this PR https://github.com/k2-fsa/icefall/pull/233
+And the SpecAugment codes from this PR https://github.com/lhotse-speech/lhotse/pull/604
+
+Conformer encoder + non-current decoder. The decoder
+contains only an embedding layer and a Conv1d (with kernel size 2).
+
+The WERs are
+
+|                                    |     dev    |    test    | comment                                  |
+|------------------------------------|------------|------------|------------------------------------------|
+|          greedy search             | 7.19       | 6.57       | --epoch 29, --avg 16, --max-duration 100 |
+|      beam search (beam size 4)     | 7.12       | 6.37       | --epoch 29, --avg 16, --max-duration 100 |
+| modified beam search (beam size 4) | 7.00       | 6.19       | --epoch 29, --avg 16, --max-duration 100 |
+
+The training command for reproducing is given below:
+
+```
+export CUDA_VISIBLE_DEVICES="0,1,2,3"
+
+./transducer_stateless/train.py \
+  --world-size 4 \
+  --num-epochs 30 \
```

Comment:
The tensorboard log in the following contains information only up to epoch-28. Can you upload the whole log?

Reply:
The following tensorboard log is the whole log. It includes 30 epochs (from 0 to 29 epoch).
![image](https://user-images.githubusercontent.com/37799481/156487200-72435a1c-f1b3-4915-b8e6-4084d21ed7dc.png)

Reply:
Sorry, I missed it.